### ISSUE TITLE
delete files or Collection failing

### ISSUE THREAD

**Issue by grabani** [h2oai/h2ogpt#1165]
Hi

How do I delete (a) entire collections or (b) remove specific files from collections.?

### REMOVE SPECIFIC FILES
When I select a single file under the **Select Subset of Document(s) for Chat with Collection: MyData** section

![image](https://github.com/h2oai/h2ogpt/assets/25469154/edab5949-6f28-4fe0-9329-00e80f7ea256)

and select the **Deleted Selected Sources from the DB** I get the error:

![image](https://github.com/h2oai/h2ogpt/assets/25469154/4c2ef3ec-e3ae-4506-86dd-e1ecef696432)

### REMOVE ENTIRE COLLECTIONS
How can I delete an entire collection (made up of several files)?

![image](https://github.com/h2oai/h2ogpt/assets/25469154/548fe9c4-9e71-480b-9bb2-99995b0cf9a3)

Note: I have uploaded for files four text files each about 80KB only.

![image](https://github.com/h2oai/h2ogpt/assets/25469154/09eaa0ae-286a-4ba2-91ba-f93c8de295dd)

**Comment by pseudotensor** [comment]
The first issue is probably due to a chromadb change, will look.

The second issue is achieved by adding the collection  name to "Purge Collection".  That deletes from UI, DB, and source files as it says.

**Comment by grabani** [comment]
@pseudotensor thanks for your response. 

With regards to the second point/question, I added what I considered the collection name and pressed “purge collection” option. But I do not see the files deleted. I’m not sure what message I should get to inform me the connection has been deleted.

Based on the screenshot I have provided can you kindly identify for me what is the “collection” name in my situation? Also, what should I see which confirms the data has been deleted. 

Thanks again.

**Comment by grabani** [comment]
I am now getting the same/similar error when trying to delete the entire collection. I determined that my  collection's name was "MyData". Furthermore, I worked out the way to delete a collection was to type in the name of my collection in the` Purge Collection (UI, DB, & source files)` field and then hit ENTER (previously I was incorrectly clicking on `Update DB with new/changed files on disk`). When I hit ENTER I get the below error:

![image](https://github.com/h2oai/h2ogpt/assets/25469154/0c2aa302-28dd-4af9-911c-dd92d63c2a04)

I then have to log back in, after which the my collection is still there (i.e. not deleted):

![image](https://github.com/h2oai/h2ogpt/assets/25469154/11ce786b-aebc-461e-bc36-8ab5bb15cfc7)
